core/store/migrations/migration1581240419/migrate.go

```diff
@@ -0,0 +1,29 @@
+package migration1581240419
+
+import (
+	"chainlink/core/store/dbutil"
+	"github.com/jinzhu/gorm"
+)
+
+// Migrate moves job_runs.overrides to run_requets.request_params
+func Migrate(tx *gorm.DB) error {
+	err := tx.Exec(`ALTER TABLE run_requests ADD request_params text NOT NULL DEFAULT '{}'`).Error
```

Comment:
Remove default here.

Reply:
Would be better to utilize jsonb for postgres? sqlite should just ignore column types (it always uses string).

Reply:
Yes JSONB would be much better. We don't use it for any of our other json columns AFAICT (although perhaps we should)